### PR TITLE
Removed unused usings.

### PR DESCRIPTION
**Pull request by CloneDeath**
As the title says, I just removed unused `using`s.
There is still an unused using in the generated SPIRV file (Vulkan). But I think we can deal with that at a future date, separately.

### CODE REVIEW

File: Ryujinx/Ui/Windows/CheatWindow.cs [hunk 1]
```diff
@@ -7,7 +7,6 @@
 using System.Linq;
 
 using GUI        = Gtk.Builder.ObjectAttribute;
```
Comment (gdkchan) [inline Ryujinx/Ui/Windows/CheatWindow.cs:9]:
```suggestion
using GUI = Gtk.Builder.ObjectAttribute;
```

**Review by gdkchan — APPROVED** [review]
lgtm, thanks.

**Review by Thog — APPROVED** [review]
A long overdue cleanup, great work 👍

**Comment by Thog** [comment]
Will need a rebase before merge

**Comment by CloneDeath** [comment]
@Thog rebased.

**Pull request merged**